src/cp/cp.js

```diff
@@ -1,6 +1,12 @@
+import { fork } from 'child_process';
+
 const spawnChildProcess = async (args) => {
-    // Write your code here
+    const childPath = new URL('files/script.js', import.meta.url);
+    const childProcess = fork(childPath, args, {
+        stdio: ['pipe', 'pipe', 'pipe', 'ipc'],
+    });
+    process.stdin.pipe(childProcess.stdin);
+    childProcess.stdout.pipe(process.stdout);
```

Comment:
Fork by default creates an IPC channel and also automatically connects all threads between the main/child processes.